programs/cleos/main.cpp

```diff
@@ -148,6 +149,22 @@ FC_DECLARE_EXCEPTION( localized_exception, 10000000, "an error occured" );
     FC_MULTILINE_MACRO_END \
   )
 
+//copy pasta from keosd's main.cpp
```

Comment:
the changes in this file are left over cruft from when unix socket was the default; probably could remove all this